src/SSL.cpp

```diff
@@ -34,6 +34,67 @@
 
 #include "Version.h"
 
+QString MumbleSSL::defaultOpenSSLCipherString() {
+	return QLatin1String("ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:kEDH+AESGCM:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA256:DHE-RSA-AES256-SHA256:DHE-DSS-AES256-SHA:DHE-RSA-AES256-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:AES:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!CAMELLIA:!MD5:!PSK:!SRP:!DH:!aECDH:!EDH-DSS-DES-CBC3-SHA:!EDH-RSA-DES-CBC3-SHA:!KRB5-DES-CBC3-SHA");
```

Comment:
AFAIK (and [citation needed] agrees), the use of SHA2 was standardised in TLS 1.2 in 2008; support when connecting with lower TLS standards is purely incidental. Due to the recent issues with TLS 1.0, standing recommendations against SHA-1 and the age of the TLS 1.2 standard, I really don't see a reason to support it out of the box.

Something like `!aNULL:!eNULL:!EXP:!LOW:!MD5:EECDH+AES:EDH+AES:ECDHE:-SHA1:-RC4:EDH+SHA:DHE+AES:EDH+RC4:RSA+SHA256:AES256-SHA:DES-CBC3-SHA` is pretty short and yields good results on the Qualys SSL Labs check even on the web when targeting a variety of legacy browsers, removing the SHA ciphers from that should do and would look more maintainable to me.